aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -12,6 +12,51 @@
 from aws_lambda_powertools.utilities.typing import LambdaContext
 
 logger = logging.getLogger(__name__)
+APPLICATION_JSON = "application/json"
+
+
+class ServiceError(Exception):
+    """Service Error"""
+
+    def __init__(self, status_code: int, msg: str):
+        """
+        Parameters
+        ----------
+        status_code: int
+            Http status code
+        msg: str
+            Error message
+        """
+        self.status_code = status_code
+        self.msg = msg
+
+
+class BadRequestError(ServiceError):
+    """Bad Request Error"""
+
+    def __init__(self, msg: str):
+        super().__init__(400, msg)
+
+
+class UnauthorizedError(ServiceError):
+    """Unauthorized Error"""
+
+    def __init__(self, msg: str):
+        super().__init__(401, msg)
+
+
+class NotFoundError(ServiceError):
+    """Not Found Error"""
+
+    def __init__(self, msg: str = "Not found"):
+        super().__init__(404, msg)
```

Comment:
Could you move these to an `exceptions.py`?

Reply:
@heitorlessa  But what about if appsync has it's own exceptions? Or do we reused them?

Reply:
AppSync uses GraphQL and has a different mechanism to handle exceptions. These errors, for example, wouldn't make back to the client as it'd in API Gateway unless customers implement them in VTL.